js/mapbox-gl.js.flow

```diff
@@ -0,0 +1,367 @@
+type Style = Object;
+type Filter = Array<any>;
+type Source = Object;
+type Layer = Object;
+
+type MapOptions = {
+  container: HTMLElement | string,
+  minZoom?: number,
+  maxZoom?: number,
+  style?: Style | string,
+  hash?: boolean,
+  interactive?: boolean,
+  bearingSnap?: number,
+  classes?: Array<string>,
+  attributionControl?: boolean,
+  failIfMajorPerformanceCaveat?: boolean,
+  preserveDrawingBuffer?: boolean,
+  maxBounds?: LngLatBoundsLike,
+  scrollZoom?: boolean,
+  boxZoom?: boolean,
+  dragRotate?: boolean,
+  dragPan?: boolean,
+  keyboard?: boolean,
+  doubleClickZoom?: boolean,
+  touchZoomRotate?: boolean,
+  trackResize?: boolean,
+  center?: Array<number>,
```

Comment:
In this and other places where we mention the `center` argument, it is technically a LngLatLike, so can either be a two-element array _or_ a LngLat object. Maybe LngLatLike should be a type alias?